File: backend/src/main/java/at/ac/tuwien/sepm/groupphase/backend/endpoint/dto/ticket/TicketQueueDto.java
package at.ac.tuwien.sepm.groupphase.backend.endpoint.dto.ticket;

import at.ac.tuwien.sepm.groupphase.backend.endpoint.dto.seat.SeatDisplayDto;
import lombok.Data;

import java.util.List;

@Data
public class TicketQueueDto {

    private int id;
    private List<SeatDisplayDto> seats;

}

File: frontend/src/app/dtos/ticket/ticket-queue-dto.ts
import { TicketSeatDto } from "./ticket-seat-dto";

export class TicketQueueDto {

  constructor(private _id: number, private _seats: TicketSeatDto[]) {
  }

  get id(): number {
    return this._id;
  }

  get seats(): TicketSeatDto[] {
    return this._seats;
  }
}
